mediacapture-streams/MediaStream-MediaElement-preload-none-src.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<!-- Copyright © 2016 Chromium authors and World Wide Web Consortium, (Massachusetts Institute of Technology, ERCIM, Keio University, Beihang). -->
+<html>
+    <head>
+        <title>Test that the HTMLMediaElement preload 'none' attribute value is ignored for MediaStream object URLs used as src.</title>>
+        <link rel="author" title="Matthew Wolenetz" href="mailto:wolenetz@chromium.org"/>
+        <script src="/resources/testharness.js"></script>
+        <script src="/resources/testharnessreport.js"></script>
+        <script src="/common/vendor-prefix.js" data-prefixed-objects='[{"ancestors":["navigator"], "name":"getUserMedia"}, {"ancestors":["window"], "name":"MediaStream"}]'></script>
+    </head>
+    <body>
+        <p class="instructions">When prompted, accept to share your audio stream.</p>
+        <h1 class="instructions">Description</h1>
+        <p class="instructions">This test checks that the HTMLMediaElement preload 'none' attribute value is ignored for MediaStream object URLs used as src.</p>
+
+        <audio id="aud"></audio>
+
+        <script>
+            var aud = document.getElementById("aud");
+            aud.preload="none";
+            var t = async_test("Tests that the HTMLMediaElement preload attribute value 'none' is ignored for MediaStream object URLs used as src.");
```

Comment:
The default title is taken from `document.title`, so this can be simplified to `async_test(function(t) { ... } )`